core/src/operator.rs

```diff
@@ -220,7 +220,7 @@ where
             // The amount will be calculated as if the transaction has 1 input
             // and (num_kickoff_utxos + 2) outputs where the first k outputs are
             // the kickoff outputs, the penultimante output is the change output,
-            // and the last output is the anyonecanpay output for fee bumping.
+            // and the last output is the anyone canpay output for fee bumping.
```

Comment:
I believe this is written bc there is a sighash type called ANYONECANPAY

Reply:
@januszgrze can you fix this?